### PR TITLE
Force future date for expiration / deletion dates

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user
NONE
```

### CODE REVIEW

**Review by petersutter — APPROVED** [review]
/lgtm

File: frontend/src/components/TimeString.vue [hunk 1]
```diff
@@ -48,9 +49,8 @@ export default {
     dateTime: {
       type: String
     },
-    pointInTime: {
-      type: Number,
-      default: 0 // negative = force past date, positive = force future date
+    mode: {
+      type: String
```
Comment (holgerkoser) [inline frontend/src/components/TimeString.vue:53]:
```suggestion
      type: String,
      default: 'past'
```

File: frontend/src/components/TimeString.vue [hunk 2]
```diff
@@ -87,9 +87,9 @@ export default {
       const currentDate = new Date().getTime()
       const refDate = new Date(dateTimeValue).getTime()
       let diffInMilliseconds
-      if (this.pointInTime < 0) {
+      if (toLower(this.mode) === 'past') {
```
Comment (holgerkoser) [inline frontend/src/components/TimeString.vue:90]:
```suggestion
      if (this.mode === 'past') {
```

File: frontend/src/components/TimeString.vue [hunk 3]
```diff
@@ -87,9 +87,9 @@ export default {
...
         this.negativeRefDate = true
-      } else if (this.pointInTime > 0) {
+      } else if (toLower(this.mode) === 'future') {
```
Comment (holgerkoser) [inline frontend/src/components/TimeString.vue:92]:
```suggestion
      } else if (this.mode === 'future') {
```

File: frontend/src/components/TimeString.vue (hunk 1 above)
Comment (grolu) [reply frontend/src/components/TimeString.vue:52]: we do not want a default here as the default is dynamic / auto detect mode.

**Review by petersutter — APPROVED** [review]

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**